smoketest/scripts/cli/test_nat.py

```diff
@@ -276,7 +278,7 @@ def test_nat_balance(self):
         self.cli_set(dst_path + ['rule', '1', 'load-balance', 'backend', member_1, 'weight', weight_1])
         self.cli_set(dst_path + ['rule', '1', 'load-balance', 'backend', member_2, 'weight', weight_2])
 
-        self.cli_set(src_path + ['rule', '1', 'outbound-interface', ifname])
+        self.cli_set(src_path + ['rule', '1', 'outbound-interface', 'interface-name', ifname])
```

Comment:
We voted for another syntax
Like “outbound-interface name|group”

should we also revert https://github.com/vyos/vyos-1x/pull/2355#issuecomment-1776514808?

Reply:
Why not bring in the new CLI syntax in 1.5 and backport it here? So we can test it already in current versions

Reply:
It just required rewriting it again.

original
`'rule', rule, 'outbound-interface', outbound_iface`

first replace 
`'rule', rule, 'outbound-interface', 'interface-group', interface_group`

second replace
`'rule', rule, 'outbound-interface', 'group', interface_group]`

Why not create a second option right away?

so if we get the second options anyway I'd approve it ;) @nicolas-fort